src/test/java/org/commcare/formplayer/tests/CasePaginationTests.java

```diff
@@ -39,11 +39,11 @@ protected String getMockRestoreFileName() {
     public void testSort() throws Exception {
         EntityListResponse entityListResponse =
                 sessionNavigate("requests/navigators/pagination_navigator.json", EntityListResponse.class);
-        assert entityListResponse.getEntities().length == EntityListResponse.CASE_LENGTH_LIMIT;
+        assert entityListResponse.getEntities().length == 20;
         String previousName = "";
         for (EntityBean entity: entityListResponse.getEntities()) {
             String currentName = (String) entity.getData()[0];
-            assert previousName.compareTo(currentName) < 0;
+            assert previousName.toLowerCase().compareTo(currentName.toLowerCase()) <= 0;
```

Comment:
`EntitySorter` converts texts values [to lowercase](https://github.com/dimagi/commcare-core/blob/master/src/main/java/org/commcare/cases/entity/EntitySorter.java#L90) before comparing, so doing the same here and taking into account for same text values.

Reply:
This would be a good in-code comment